### PR TITLE
Winter 2021 update

### PR DESCRIPTION
**Pull request by k8hertweck**
Hopefully this is ready to go live on Monday.

### CODE REVIEW

**Comment by k8hertweck** [comment]
@lakikowolfe give this a proofreading for typos, clarity, etc. I'm happy to make corrections, otherwise publish when you're ready!

File: _posts/2021-02-23-winter-update.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+We've recognized the way that people value and interact with data
+has shifted with so much of our community working remotely,
+and have adjusted our methods of supporting training to match these approaches.
+
+While the materials we use for teaching have always been publibly available (online through GitHub repositories),
+we converted our classes to accessible, fully-rendered websites
+that are available for reference during class,
+as well as for self-led, work-at-your-own-pace learning.
+These materials include:
+- [Intro to R](https://github.com/fredhutchio/r_intro)
```
Comment (lakikowolfe) [inline _posts/2021-02-23-winter-update.md]: just wondering if it makes sense to link to the fully-rendered website? If not, maybe we should make the link to the website more visible in the README. I don' think a lot of folks think to look for the link on the sidebar.

Comment (k8hertweck) [reply _posts/2021-02-23-winter-update.md]: You're absolutely right; can you update those links? I thought those were the ones I was grabbing

Comment (lakikowolfe) [reply _posts/2021-02-23-winter-update.md]: yup! on it!

**Pull request merged**